qa/L0_model_update/instance_update_test.py

```diff
@@ -457,15 +467,116 @@ def test_instance_resource_decrease(self):
                 # explicit limit of 10 is set.
                 self.assertNotIn("Resource: R1\t Count: 3", f.read())
 
-    # Test for instance update on direct sequence scheduling
-    @unittest.skip("Sequence will not continue after update [FIXME: DLIS-4820]")
-    def test_instance_update_on_direct_sequence_scheduling(self):
-        pass
+    # Test sequence scheduler update
+    def test_sequence_instance_update(self):
```

Comment:
Add high level description on how the test operates

Reply:
Yes. Added more descriptions on the scheduler update test, on [Add more documentaion for scheduler update test](https://github.com/triton-inference-server/server/pull/5831/commits/c070b68e8c4a4884f0cbc9c556e0bd66684c1885) commit.